tests/files/test-vip-filesystem.php

```diff
@@ -386,4 +386,23 @@ public function data_get_transport_for_path(): iterable {
 			[ constant( 'WP_CONTENT_DIR' ) . '/languages/test.txt', 'direct' ],
 		];
 	}
+
+	public function test_wp_font_dir() {
+		// Only available in WP 6.5 and newer:
+		if ( ! function_exists( '\wp_get_font_dir' ) ) {
+			$this->assertEquals( true, true );
+			return;
+		}
+
+		$font_dir = \wp_get_font_dir();
```

Comment:
We need to follow this PR: https://github.com/WordPress/wordpress-develop/pull/6211

In case the function is renamed/changed. (Or test it by executing the filter instead.)

Reply:
> We need to follow this PR: [WordPress/wordpress-develop#6211](https://github.com/WordPress/wordpress-develop/pull/6211)
> 
> In case the function is renamed/changed. (Or test it by doing the filter instead.)

I'll try to remember to come back and let you know.

I've been doing some expreiments on my local build and don't intend to change the name of the function or the filter but will confirm.

Reply:
Thanks!